main.py

```diff
@@ -1,10 +1,75 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
-def main():
-    pass
+def main() -> None:
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for name, data in players.items():
+
+        race = create_race(data["race"])
+        create_skill_from_list(data["race"]["skills"], race)
+
+        guild = None
+        if data["guild"] is not None:
+            guild = create_guild(data["guild"])
+
+        create_player(
+            name,
+            data,
+            race,
+            guild
+        )
+
+
+def create_player(
+        player_name: str,
+        player: dict,
+        race: int,
+        guild: int,
+) -> None:
+    if len(Player.objects.filter(nickname=player_name)) == 0:
```

Comment:
You may simplify this check using `Model.objects.filter(some_field='data').exists()` method. But I'm not sure if this is an issue.